src/iap/shared/services/environment.service.ts

```diff
@@ -0,0 +1,11 @@
+import { Inject, Injectable, Optional } from '@angular/core';
+import { ENVIRONMENT } from '@shared/constants/tokens';
```

Comment:
:point_right: :point_left: 
```suggestion
import { Inject, Injectable, Optional } from '@angular/core';

import { ENVIRONMENT } from '@shared/constants/tokens';
```